backend/sql/initial_schema.sql

```diff
@@ -0,0 +1,100 @@
+create table project
+(
+    project_id  bigserial     not null primary key,
+    name        varchar(256)  not null unique,
+    description varchar(1000) null
+);
+
+create table test
+(
+    test_id bigserial    not null primary key,
+    name    varchar(256) not null unique
+);
+
+create table subtest
```

Comment:
once again, why? if with project it could make sense but tests are really domain specific and they told as explicitly that there is a test and version of test inside (which refer to their confluence). how we should handle case when there is test inside test inside test in frontend? how it's aligned with their domain? Where confluence link should refer to in this case?

I am strongly against doing some general things if they are not required in the domain because this generality causes a lot of corner cases which if happen should be somehow properly handled on both frontend and backend. @gromakovsky what do you think?

Reply:
@pva701 because SQL migrations are a pain in the ass, significantly more of a pain in the ass than adding one extra field.

Also, I hope that subtests are a mistake and we'll convince them to use a tag (trait) system for test types. I would go as far as to do the following (if you want easy migrations):

*Just use 1-deep tables* and then, when needed, just *add* another field called `parent_id` which will give you a *free* migration!